Quality.md

```diff
@@ -17,13 +17,13 @@
 ### I ensure Octopus teams can ship with confidence
 
 - I am an advocate for quality.
-- I help engineers refine requirements and identify the right level of testing
+- I help engineers refine requirements and identify the optimal kind, level, and timing for testing.
 - I use customer’s feedback and data to measure our level of quality.
 - I identify gaps in the quality processes and collaborate with the team to close those gaps.
```

Comment:
I identify gaps in `quality processes and engineering practices`. 
Thought process: since quality is tightly coupled with engineering efforts and outputs, a change in the quality process will change our engineering practice?